python/astro_metadata_translator/headers.py

```diff
@@ -353,6 +353,12 @@ class or else support automatic translation class determination.
         modified = _find_from_resource(header, *translator.resource_root(), target_file)
 
     # Allow a translation class to do local fixups
-    translator_modified = translator_class.fix_header(header)
+    # Allow it to fail but log the failure
+    try:
+        translator_modified = translator_class.fix_header(header)
+    except Exception as e:
+        log.fatal("Ignoring translator header fixup of %s %s: %s",
+                  instrument, obsid, str(e))
```

Comment:
Is explicitly casting `e` required, since it's implicit in the format string?

Reply:
Will check.